sdk/communication/AzureCommunication/Source/Generated/CommunicationError.swift

```diff
@@ -0,0 +1,82 @@
+// --------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// --------------------------------------------------------------------------
+
+import AzureCore
+import Foundation
+// swiftlint:disable superfluous_disable_command
+// swiftlint:disable identifier_name
+// swiftlint:disable line_length
+// swiftlint:disable cyclomatic_complexity
+
+/// The Communication Services error.
+public final class CommunicationError: Codable {    
```

Comment:
If this is an error, it should also conform to the Error protocol.

Reply:
This may have been moved in by accident from the rest api generated folder. @sacheu @DominikMe @RezaJooyandeh do we need this file?

Reply:
I do not think we need it right now. You can remove it form your PT. But would be good to know about the discrepancy with the Error protocol so we can fix things when it comes to it